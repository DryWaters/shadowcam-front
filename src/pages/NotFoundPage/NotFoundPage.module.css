.notFoundBackground {
    background: url("../../assets/images/boxer-404.jpg") no-repeat center center fixed;
    background-color: black;
    height: 750px;
}

.notFoundContainer {
    align-items: center;
    display: flex;
    height: 50%;
    flex-direction: column;
    justify-content: space-between;
    text-align: center;
    width: 100%;
}

.errorHeader {
    font-size: 100px;
    font-weight: bolder;
}