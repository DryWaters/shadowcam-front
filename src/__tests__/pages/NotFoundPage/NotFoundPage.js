import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from '../../../pages/NotFoundPage/NotFoundPage'

describe("NotFoundPage render", () => {
  it("Should render NotFoundPage component without crashing", () => {
    shallow(<NotFoundPage />);
  });

  it("Should match NotFoundPage component snapshot", () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});