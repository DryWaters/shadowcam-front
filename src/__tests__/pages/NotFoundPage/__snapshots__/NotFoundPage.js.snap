// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage render Should match NotFoundPage component snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <div
        className="notFoundBackground"
      >
        <div
          className="notFoundContainer"
        >
          <div>
            <p
              className="errorHeader"
            >
              Error 404
            </p>
          </div>
          <div>
            <h2>
              Oops! It looks like this page was knocked out!
            </h2>
            <h4>
              Please check the URL and try again.
            </h4>
          </div>
        </div>
      </div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="notFoundContainer"
        >
          <div>
            <p
              className="errorHeader"
            >
              Error 404
            </p>
          </div>
          <div>
            <h2>
              Oops! It looks like this page was knocked out!
            </h2>
            <h4>
              Please check the URL and try again.
            </h4>
          </div>
        </div>,
        "className": "notFoundBackground",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <p
                className="errorHeader"
              >
                Error 404
              </p>
            </div>,
            <div>
              <h2>
                Oops! It looks like this page was knocked out!
              </h2>
              <h4>
                Please check the URL and try again.
              </h4>
            </div>,
          ],
          "className": "notFoundContainer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <p
                className="errorHeader"
              >
                Error 404
              </p>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Error 404",
                "className": "errorHeader",
              },
              "ref": null,
              "rendered": "Error 404",
              "type": "p",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                  Oops! It looks like this page was knocked out!
                </h2>,
                <h4>
                  Please check the URL and try again.
                </h4>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Oops! It looks like this page was knocked out!",
                },
                "ref": null,
                "rendered": "Oops! It looks like this page was knocked out!",
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Please check the URL and try again.",
                },
                "ref": null,
                "rendered": "Please check the URL and try again.",
                "type": "h4",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <div
          className="notFoundBackground"
        >
          <div
            className="notFoundContainer"
          >
            <div>
              <p
                className="errorHeader"
              >
                Error 404
              </p>
            </div>
            <div>
              <h2>
                Oops! It looks like this page was knocked out!
              </h2>
              <h4>
                Please check the URL and try again.
              </h4>
            </div>
          </div>
        </div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="notFoundContainer"
          >
            <div>
              <p
                className="errorHeader"
              >
                Error 404
              </p>
            </div>
            <div>
              <h2>
                Oops! It looks like this page was knocked out!
              </h2>
              <h4>
                Please check the URL and try again.
              </h4>
            </div>
          </div>,
          "className": "notFoundBackground",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                <p
                  className="errorHeader"
                >
                  Error 404
                </p>
              </div>,
              <div>
                <h2>
                  Oops! It looks like this page was knocked out!
                </h2>
                <h4>
                  Please check the URL and try again.
                </h4>
              </div>,
            ],
            "className": "notFoundContainer",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <p
                  className="errorHeader"
                >
                  Error 404
                </p>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Error 404",
                  "className": "errorHeader",
                },
                "ref": null,
                "rendered": "Error 404",
                "type": "p",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2>
                    Oops! It looks like this page was knocked out!
                  </h2>,
                  <h4>
                    Please check the URL and try again.
                  </h4>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Oops! It looks like this page was knocked out!",
                  },
                  "ref": null,
                  "rendered": "Oops! It looks like this page was knocked out!",
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Please check the URL and try again.",
                  },
                  "ref": null,
                  "rendered": "Please check the URL and try again.",
                  "type": "h4",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
